README.md

```diff
@@ -3,6 +3,18 @@
 This repo contains an initial set of cluster components to be installed and
 configured by [eksctl](https://eksctl.io) through GitOps.
 
+## Components
+
+- ALB ingress controller -- to easily expose services to the World.
+- [Cluster autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) -- to [automatically add/remove nodes](https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-autoscaler-setup/) to/from your cluster based on its usage.
+- [Helm](https://helm.sh/) -- Kubernetes' package manager.
+- [Flux](https://docs.fluxcd.io/en/stable/) and its [Flux-Helm operator](https://github.com/fluxcd/helm-operator-get-started) -- for [GitOps](https://github.com/weaveworks/awesome-gitops), i.e. git-based continuous delivery.
```

Comment:
Flux and Helm are installed as part of the `eksctl install flux` / `eksctl gitops apply`, they are not strictly part of the profile, so I am not sure we should be mentioning them here.

Reply:
Fair enough. I'll remove them. 👍